metals/src/main/scala/scala/meta/internal/metals/MetalsLanguageServer.scala

```diff
@@ -1483,33 +1488,56 @@ class MetalsLanguageServer(
   }
 
   private def autoConnectToBuildServer(): Future[BuildChange] = {
-    for {
-      _ <- disconnectOldBuildServer()
-      maybeBuild <- timed("connected to build server") {
-        if (buildTools.isBloop) bloopServers.newServer(userConfig)
-        else bspServers.newServer()
-      }
-      result <- maybeBuild match {
-        case Some(build) =>
-          connectToNewBuildServer(build)
-        case None =>
-          Future.successful(BuildChange.None)
-      }
-      _ = {
-        treeView.init()
+    def compileAllOpen: BuildChange => Future[BuildChange] = {
```

Comment:
could be `compileOpenFiles` at first I was not 100% if it meant open files.

Reply:
changed to `compileAllOpenFiles`